components/checkout/pay/SelectedPlanOverview.vue

```diff
@@ -1,57 +1,54 @@
 <script setup lang="ts">
-import type { CryptoPayment, SelectedPlan } from '~/types';
+import type { PlanParams } from '~/composables/plan';
+import type { SelectedPlan } from '~/types';
 import { get, set } from '@vueuse/core';
+import { formatDate } from '~/utils/date';
 import { getPlanNameFor } from '~/utils/plans';
 
 const props = withDefaults(
   defineProps<{
-    plan: SelectedPlan | CryptoPayment;
-    crypto?: boolean;
+    plan: SelectedPlan;
     warning?: boolean;
     disabled?: boolean;
+    nextPayment?: number;
   }>(),
   {
-    crypto: false,
     warning: false,
   },
 );
 
-const { plan } = toRefs(props);
+const { plan, nextPayment } = toRefs(props);
 const router = useRouter();
 
 const selection = ref(false);
 
-const name = computed(() => getPlanNameFor(get(plan).months));
-const date = computed(() => {
-  const currentPlan = get(plan);
-  const date = new Date(currentPlan.startDate * 1000);
-  return date.toLocaleDateString();
+const name = computed(() => {
+  const selectedPlan = get(plan);
+  return `${getPlanNameFor(selectedPlan)} - € ${selectedPlan.price.toFixed(2)}`;
```

Comment:
I guess you are aware of this build error, also we should try for refs and computed as much as possible to explicitly add types instead of leading it to infer the type.